### ISSUE TITLE
GLES 3 all right there is a reason LOD not optimizing for Camera FPS drops dramatically

### ISSUE THREAD

**Issue by ilyafuu** [godotengine/godot#31530]
![image](https://user-images.githubusercontent.com/17591539/63420097-aea2c580-c40e-11e9-95c9-93fc0726a629.png)
TPS Demo `vertices drawn 19Million` it's very bad FPS  we need LOD low :(

![image](https://user-images.githubusercontent.com/17591539/63419873-3b00b880-c40e-11e9-9b87-cad82a0896cc.png)
https://www.youtube.com/watch?v=ifNyVS2_6f8&t

https://www.reddit.com/r/godot/comments/atpjo7/simple_lod_example_project/

it's not, you need official LOD Camera support

**Comment by nyenye** [comment]
I too hope that a native LOD implementation will be available once the Vulkan rewrite is complete with Godot 4.0.

Thanks for all the hard work!

**Comment by Zylann** [comment]
Not just LOD, occlusion.

**Comment by Calinou** [comment]
Duplicate of #8806.
